addons/point_of_sale/static/src/js/screens.js

```diff
@@ -1182,7 +1182,7 @@ var ClientListScreenWidget = ScreenWidget.extend({
         
         var fields = {};
         this.$('.client-details-contents .detail').each(function(idx,el){
-            fields[el.name] = el.value;
+            fields[el.name] = el.value || false;
```

Comment:
I would perhaps use `undefined` instead of `false`. Or I think you could put an if test around `fields[el.name] = el.value;` so you don't include fields that are not set in the `fields` object.